students/seunghun/users/views.py

```diff
@@ -1 +1,43 @@
+from django.shortcuts import render
+from django.views import View
+from django.http import JsonResponse
+from users.models import User
+import json
+import re
+
+class User_CreateView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            data['email'] == "" or data['password'] == ""
+        except KeyError:
+            return JsonResponse({'MESSAGE':'KEY_ERROR'}, status = 400)
+
+        '''if data['email'] == None or data['password'] == None:
+            return JsonResponse({'MESSAGE':'KEY_ERROR'}, status = 400)'''
+
+        if User.objects.filter(email = data['email']).exists():
+            return JsonResponse({'MESSAGE':'USE_OTHER_EMAIL'}, status = 400)
+        
+        email_check = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+        emails = data['email']
+        if email_check.match(emails) == None:
+            return JsonResponse({'MESSAGE':'CHECK_EMAIL_FOAM'}, status = 400)
+        
+        password_check = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
+        passwords = data['password']
```

Comment:
```suggestion
        password = data['password']
```
항상 하나의 데이터만 들어오니 단수로 지정해주세요.

Reply:
넵!